### ISSUE TITLE
Multiple block damage values for the same default leaf block

### ISSUE THREAD

**Issue by xkforce** [ForestryMC/ForestryMC#2206]
Basically the issue is that there are multiple block damage values that correspond to the same leaf block.  eg. forestry:leaves.decorative.0:6 are what you get if you pick block Hill Cherry leaves.  forestry:leaves.default.0:6 should be its equivalent that can decay but it's not, forestry:leaves.default.0:6 is Silver Birch.  But forestry:leaves.default.0:2 is also Silver Birch.  If you pick block forestry:leaves.default.0:2 you get forestry:leaves.decorative.0:2 as expected.

**Comment by temp1011** [comment]
Pick block for leaves is a bit weird. Just using `/give` for `forestry:leaves.default` doesn't give a leaf block with a species since it needs some extra data too. Some leaves may also have extra vanilla variants too.

**Comment by xkforce** [comment]
forestry.decorative is what you get with pick block.  forestry.default is a set of blocks that are similar to the decorative variety but can decay but most leaf blocks aren't represented in this class of blocks.  forestry.leaves is a tile entity with nbt data tacked on to it which defines each leaf block that spawns naturally and in leaf blocks generated when a tree grows.

There is no means of controlling what trees do or do not spawn from within the mod.  Which means that if you want certain trees to spawn naturally, you'll have to add them yourself.  But... the way Forestry is designed basically makes it impossible to spawn in trees artificially.  The nbt data isn't easily extracted from the tree leaf blocks and even if it was, the tree generation mods that I know of don't support tacking on the necessary nbt data on to the forestry.leaves tile entity block.  Even if it did, trees do not properly drop fruit unless they've been bred.

So basically the way it is, it isn't possible to incorporate forestry into a pack that was meant to add more variety but not require the player to essentially memorize a genetic circuit diagram to get any of the trees that don't spawn naturally.

So I guess the real underlying problem is that forestry is basically a nuclear reactor.  Which is fine for those that want it but there's no way to use forestry like any other tree or crop mod.  i.e go exploring and find a tree and grab some fruit or saplings to bring back to the base to start a simple farm.

**Comment by temp1011** [comment]
I can implement configurable worldgen for tree species but ideally I will need https://github.com/ForestryMC/ForestryMC/pull/2197. I will add it to the other worldgen stuff I am working on.

**Comment by xkforce** [comment]
Thank you.  I've been wanting to integrate Forestry a bit more cleanly into my pack and that would help a lot.
